docs/function/handlers.md

```diff
@@ -137,6 +137,24 @@ class Handler extends SqsHandler
 return new Handler();
 ```
 
+### Partial Batch Response
+
+While handling a batch of records, you can mark it as partially successful to reprocess only the failed records.
```

Comment:
That will only work if users have enabled partial batch responses on their apps though. With Lift [eventually](https://github.com/getlift/lift/pull/145) that will be automatic, but I think here we should mention that at least?

Reply:
Yeah, this is a first proposal made by @afu-dev (thanks by the way).

I agree we should add information for the config required to use this and maybe also mention that when using Lift this is enabled by default once the related PR is merged.

I'll try to get everything ready next week.

Reply:
👍  no problem, sounds great!

Reply:
Good catch, I didn't know I had to declare a `functionResponseType` in the serverless file.

I'll send @t-richard another PR